operations.js

```
@@ -0,0 +1,42 @@
+// Returns addition of two or n numbers
+exports.addition = function (args) {
+    var result = 0;
+    args.forEach(element => {
+        result += element;
+    });
+    return result;
+};
+
+// Returns difference of two numbers
+exports.subtraction = function (args) {
+    if (args.length > 2) {
+        return "can take only 2 arguments";
+    }
+    return args[0] - args[1]
+};
+
+// Returns product of two or n numbers
+exports.multiplication = function (args) {
+    var result = 1;
+    args.forEach(element => {
+        result *= element;
+    });
+    return result;
+};
+
+//Returns the quotient after the division of two numbers
+exports.division = function (args) {
+    if (args.length > 2) {
+        return "can take only 2 arguments";
```

should throw an error rather than returning